{{cookiecutter.project_slug}}/requirements/base.txt

```diff
@@ -25,6 +25,7 @@ flower==0.9.5  # https://github.com/mher/flower
 {%- endif %}
 {%- if cookiecutter.use_async == 'y' %}
 uvicorn==0.12.1  # https://github.com/encode/uvicorn
+wsproto==0.15.0  # https://github.com/python-hyper/wsproto/
```

Comment:
Good catch, I believe it's because of https://github.com/encode/uvicorn/pull/666. A bit of a shame that there is no `extra_require` for it from `uvicorn`. I guess we'll have to pay attention to `uvicorn` updates in case they remove the integration or provide an extra for it.

Didn't know about this library, looks good.